home01/js/script_4.js

```diff
@@ -0,0 +1,4 @@
+// 4.Запросите у пользователя радиус окружности и выведите площадь такой окружности.
+const PI = 3.1415;
+const circleRadius = +prompt(`Enter circle radius:`);
+alert(`Area of a circle: ${PI * (circleRadius ** 2)}`);
```

Comment:
можно без скобок, у оператора `**` приоритет выше
```suggestion
alert(`Area of a circle: ${PI * circleRadius ** 2}`);
```